### DISCUSSION TITLE
Can't establish DAVdroid sync

### DISCUSSION THREAD

**Discussion by vovpff** [YetiForceCompany/YetiForceCRM#14387] [How to] [answered]
I tried DAV feature without any result.

There is error from DAVdroid app

```
2021-03-09 16:57:27 16273 [HttpClient] <?xml version="1.0" encoding="utf-8"?>
<d:error xmlns:d="DAV:" xmlns:s="http://sabredav.org/ns">
  <s:sabredav-version>3.2.2</s:sabredav-version>
  <s:exception>Sabre\DAV\Exception\NotAuthenticated</s:exception>
  <s:message>No 'Authorization: Digest' header found. Either the client didn't send one, or the server is misconfigured. Login was needed for privilege: {DAV:}read on </s:message>
</d:error>

2021-03-09 16:57:27 16273 [HttpClient] <-- END HTTP (403-byte body)
```

Also I trier to reach /dav.php as presented on video. After login nothing happens. Logs clear.

How to understand what is wrong?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
what do you have apache or nginx? Dav requires rules, e.g.
https://github.com/YetiForceCompany/YetiForceCRM/blob/907d5257d9b1fb3eb4370189ec989e9e8b02870d/.htaccess#L8-L23
https://github.com/YetiForceCompany/YetiForceCRM/blob/907d5257d9b1fb3eb4370189ec989e9e8b02870d/tests/setup/nginx/yetiforce.conf#L40-L50
and enabled configuration: config\Api.php
![image](https://user-images.githubusercontent.com/10198654/110747028-02b60a80-823e-11eb-9c52-87a7dfb9eb4a.png)

debugging: config\Debug.php
![image](https://user-images.githubusercontent.com/10198654/110747180-401a9800-823e-11eb-9cdb-ff3791907c7b.png)

**Reply by vovpff** [reply]
@mariuszkrzaczkowski 

If I add this to `/public_html/.htaccess`

```
<IfModule mod_rewrite.c> 
 	Options +FollowSymLinks 
 	RewriteEngine On 
 	RewriteRule ^favicon.ico layouts/basic/images/favicon.ico [L,NC] 
 ...
	RewriteRule .* - [E=HTTP_AUTHORIZATION:%{HTTP:Authorization},L] 
 	RewriteRule .* - [E=HTTP_CONTENT_TYPE:%{HTTP:Content-Type},L] 
 </IfModule>
```

I get Internal Server Error

the same code in `/.htaccess` cause no errors but I cant login to dav.php

**Reply by mariuszkrzaczkowski** [reply]
logs

**Reply by vovpff** [reply]
Solved. This is some kind of apache2 configuration malfunction.

@mariuszkrzaczkowski Now sync with DAVdroid working but calendar sync only "Meeting" event types. Is it normal condition?

**Reply by mariuszkrzaczkowski** [reply]
yes , look 
https://www.davx5.com/faq/tasks/advanced-task-features  https://www.reddit.com/r/androidapps/comments/fi6i1b/tasktodo_app_that_syncs_via_davx5_davdroid/

use for application tasks OpenTasks or Tasks   https://doc.yetiforce.com/apps/
